Source/JavaScriptCore/runtime/ISO8601.cpp

```diff
@@ -530,10 +530,38 @@ std::optional<int64_t> parseUTCOffsetInMinutes(StringView string)
 template<typename CharacterType>
 static bool canBeCalendar(const StringParsingBuffer<CharacterType>& buffer)
 {
-    // https://tc39.es/proposal-temporal/#prod-Calendar
-    // Calendar :
-    //     [u-ca= CalendarName]
-    return buffer.lengthRemaining() >= 6 && buffer[0] == '[' && buffer[1] == 'u' && buffer[2] == '-' && buffer[3] == 'c' && buffer[4] == 'a' && buffer[5] == '=';
+    // https://tc39.es/proposal-temporal/#sec-temporal-parseisodatetime
```

Comment:
This is not corresponding to the syntax definition section. Please use a link to that.
Also, please describe the BNF form of this as the same to the other function (like, `parseTimeSpec` for example).

Reply:
Fixed in [19227db](https://github.com/WebKit/WebKit/pull/36361/commits/19227db637e5c5b34d4676a07f7a0147f9f991fa)

Reply:
Please check `parseTimeSpec` comment carefully. We do not need to have many URLs, but we need documentation comment for parsing logic.